book/visual-effects.md

```diff
@@ -1503,6 +1319,138 @@ are almost endless.
 
 [overflow-prop]: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow
 
+Optimizing Surface Use
+======================
+
+While the `mask` CSS property is relatively uncommon used (as is `clip-path`
+actually), there is a special kind of mask that is very common: rounded
+corners. Now that we know how to implement masks, this one is also easy to
+add to our browser (draw a mask image with the `drawRRect` Skia canvas method).
+But because it's so common, Skia provides a special-purpose
+method to clip to rounded corners: `clipRRect`.
+
+Rounded corners are specified in CSS via `border-radius`. Here's an example:
+
+    <div style="width:256px; height:256px;
```

Comment:
Change to not use width or height, but instead text, to make it work in our browser?

Reply:
Yes. I haven't passed over this section yet but definitely switch to text.